x/oracle/types/task.go

```diff
@@ -139,7 +162,7 @@ func (t *TxTask) GetValidTime() (int64, time.Time) {
 }
 
 func (t *TxTask) IsValid(ctx sdk.Context) bool {
-	return !t.ValidTime.Before(ctx.BlockTime())
+	return t.ValidTime.After(ctx.BlockTime())
```

Comment:
we tend to consider task is valid if `BlockTime <= validTime`.
In original implementation, this logic aren't consistent. Somewhere it's `BlockTime < validTime`, in other place it's `BlockTime <= validTime`

Reply:
fixed